examples/gettingStarted/scheduledMessages.ts

```diff
@@ -0,0 +1,103 @@
+/*
+  This sample demonstrates how .scheduleMessage() function can be used to schedule messages to appear
+  on a Service Bus entity at a specified later time.
+*/
+
+import {
+  Namespace,
+  SendableMessageInfo,
+  OnMessage,
+  MessagingError,
+  OnError,
+  ServiceBusMessage
+} from "../../lib";
+import { delay } from "rhea-promise";
+
+// Define connection string and related Service Bus entity names here
+const connectionString = "";
+const queueName = "";
+
+let ns: Namespace;
+
+const listOfScientists = [
+  { name: "Einstein", firstName: "Albert" },
+  { name: "Heisenberg", firstName: "Werner" },
+  { name: "Curie", firstName: "Marie" },
+  { name: "Hawking", firstName: "Steven" },
+  { name: "Newton", firstName: "Isaac" },
+  { name: "Bohr", firstName: "Niels" },
+  { name: "Faraday", firstName: "Michael" },
+  { name: "Galilei", firstName: "Galileo" },
+  { name: "Kepler", firstName: "Johannes" },
+  { name: "Kopernikus", firstName: "Nikolaus" }
```

Comment:
I was wondering if we are gaining anything by having the first and last name split like this and then joining them together when creating the message body.

Why not just have `listOfScientists` be an array of string?

Reply:
Maybe using JSON representation is a little more closer to real world examples? Either way, I'm okay with to update.
Also, just noticed the attributes could be better named, updated these.